spec/mailers/user_mailer_spec.rb

```diff
@@ -432,4 +432,74 @@
       end
     end
   end
+
+  describe "invited to collection notification" do
```

Comment:
Could you include the underscores here? I believe the other tests do, since we try to use method names.

Reply:
Yes